app/api/resources/mentorship_relation.py

```diff
@@ -335,3 +336,137 @@ def put(cls, request_id, task_id):
         response = TaskDAO.complete_task(user_id=user_id, mentorship_relation_id=request_id, task_id=task_id)
 
         return response
+
+@mentorship_relation_ns.route('mentorship_relation/<int:request_id>/task/<int:task_id>/comment')
+class CreateComment(Resource):
+
+    @classmethod
+    @jwt_required
+    @mentorship_relation_ns.doc('create_comment_for_task')
+    @mentorship_relation_ns.expect(auth_header_parser, create_comment_request_body)
+    @mentorship_relation_ns.response(200, '%s'%messages.COMMENT_WAS_CREATED_SUCCESSFULLY)
+    @mentorship_relation_ns.response(400, '%s'%messages.COMMENT_FIELD_IS_MISSING)
+    @mentorship_relation_ns.response(401, '%s'%messages.USER_NOT_INVOLVED_IN_THIS_MENTOR_RELATION)
+    @mentorship_relation_ns.response(404, '%s\n%s'%(
+        messages.TASK_DOES_NOT_EXIST, messages.MENTORSHIP_RELATION_DOES_NOT_EXIST
+        ))
+    def post(cls, request_id, task_id):
+        """
+        Add a comment for a task.
+        """
+        # TODO check if user id is well parsed, if it is an integer
```

Comment:
is there anything left TODO?

Reply:
Most endpoints in resources/mentorship_relation.py have this comment. I guess some user_id check will be added in future, so I left the comment in my endpoints as well.

Reply:
I do not know what it'll actually be used for. Should I remove it or let it be?

Reply:
I removed the comment. The work is done by get_jwt_token() function.